### DISCUSSION TITLE
How to get the paper title and abstract of paper nodes in citation graph dataset such as Cora and Ctieseer?

### DISCUSSION THREAD

**Discussion by jiaruHithub** [pyg-team/pytorch_geometric#4489] [Q&A]
Hi, how to get the paper title and abstract of paper nodes in citation graph dataset such as Cora and Ctieseer?
Thanks for your anwser!

**Comment by rusty1s** [comment]
The Planetoid datasets are provided in https://github.com/kimiyoung/planetoid. As far as I can see, you can find some information about the raw datasets there. There also exist some issues and further discussion related to this. Hope this is helpful :)
